LICENSE.md

```diff
@@ -0,0 +1,65 @@
+# LICENSE
+
+The Turing Way Community make all their materials publicly available under open source licenses.
+
+The process documents and data are made available under a CC-BY license.
+Software are made available under an MIT license.
+
+The text below is re-used, under a CC-BY license from The Carpentries community materials.
```

Comment:
This confused me a bit. One light suggestion could be:

```suggestion
The  license text listed below (describing both CC-BY and MIT licenses as well as their usage in The Turing Way) is re-used under a CC-BY license from The Carpentries community materials.
```

Or something like that !